Source/CompatibilityPatches/Multiplayer.cs

```diff
@@ -0,0 +1,69 @@
+﻿using HarmonyLib;
+using Multiplayer.API;
+using PrisonLabor.Core.BaseClasses;
+using PrisonLabor.Core.BillAssignation;
+using PrisonLabor.Core.MainButton_Window;
+using PrisonLabor.Core.Other;
+using PrisonLabor.Core.Trackers;
+using RimWorld;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using UnityEngine;
+using Verse;
+
+namespace PrisonLabor.CompatibilityPatches
+{
+    [StaticConstructorOnStartup]
+    public static class Multiplayer
```

Comment:
Just wonder. This approach requires to always release mod with Multiplayer.dll or have that one on active mod list.

Do you think it will work if you do something like here: https://github.com/Aviuz/PrisonLabor/blob/master/Source/CompatibilityPatches/Quarry.cs as check if MP is enabled and just included in project references?

Reply:
not sure what you mean by Multiplayer.dll, do you mean the actual Multiplayer mod itself? because you don't need to include the mod here or in the active mod list, only the MPAPI dll is required